### DISCUSSION TITLE
Find-TppCertificate -path "\VED\Policy\" -Template "TemplateName" -Recursive

### DISCUSSION THREAD

**Discussion by TonyJHART** [Venafi/venafips#104] [Ideas]
Would love to have a Template parameter that could filter based on Template used to create certificate.
Find-TppCertificate -path "\VED\Policy\" -Template "TemplateName" -Recursive

**Comment by gdbarron** [comment]
Hi @TonyJHART.  You can use the following to search based on template, `Find-TppObject -Pattern '*\TemplateName' -Attribute 'Certificate Authority'`.  The Certificate Authority attribute stores the full path to the template, eg. \VED\Policy\Administration\CAs\TemplateName, so the pattern uses a wildcard for the path, or you could provide the full path.
